src/general/GameProperties.cs

```diff
@@ -54,6 +54,8 @@ public static GameProperties StartNewMicrobeGame(bool freebuild = false)
             game.TutorialState.Enabled = false;
         }
 
+        SaveHelper.ResetSaveFreshness();
```

Comment:
I don't think this should go here, This should be in the MainMenu when the scene is entered (or alternatively on both the new game and freebuild buttons).